src/l500/l500-options.cpp

```diff
@@ -226,6 +226,8 @@ namespace librealsense
 
     void l500_options::on_set_option(rs2_option opt, float value)
     {
+        verify_max_usable_range_restrictions(value);
```

Comment:
`opt` may not be `VISUAL_PRESET`... but inside the `verify` fn you compare the value to `MAX_RANGE`...? Am I misunderstanding?